### PR TITLE
fix wrong gpgkey fingerprint for sdl8

### PR DESCRIPTION
**Pull request by mbussolotto**
## What does this PR change?

The command spacewalk-common-channels fails on creating sdl8 channel, cause of a typo in the gpg key fingerprint. The PR fix the typo.

## GUI diff

No difference.

Before:

After:

- [ ] **DONE**

## Documentation
- No documentation needed: **add explanation. This can't be used if there is a GUI diff**
- No documentation needed: only internal and user invisible changes
- Documentation issue was created: [Link for SUSE Manager contributors](https://github.com/SUSE/spacewalk/issues/new?template=ISSUE_TEMPLATE_DOCUMENTATION.md&labels=documentation&projects=SUSE/spacewalk/31), [Link for community contributors](https://github.com/uyuni-project/uyuni-docs/issues/new).
- (OPTIONAL) [Documentation PR](https://github.com/uyuni-project/uyuni-docs/pulls)

- [ ] **DONE**

## Test coverage
- No tests: **add explanation**
- No tests: already covered
- Unit tests were added
- Cucumber tests were added

- [ ] **DONE**

## Links

Fixes #
Tracks # **add downstream PR, if any**

- [ ] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by juliogonzalez — COMMENTED** [review]
Thank you for the contribution!

Can you add a changelog entry to `spacewalk-utils.changes`?

No need for the header with date, time, etc.

Something like:

```
- Fix the GPG key finger print for Springdale Linux 8
```
is enough.

**Comment by mbussolotto** [comment]
Thanks Julio for your review! Now it should be ok

File: utils/spacewalk-utils.changes [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Fix the GPG key finger print for Springdale Linux 8
```
Comment (juliogonzalez) [inline utils/spacewalk-utils.changes:1]:
```suggestion
- Fix the GPG key fingerprint for Springdale Linux 8
```

**Comment by juliogonzalez** [comment]
I fixed a small typo at the changelog. Other than that, looks good! :-)

Thank you for the contribution, I am merging this now, and the fix will be part of the next version (most likely 2021.07).

**Pull request merged**